docs/01-app/01-getting-started/07-fetching-data.mdx

```diff
@@ -528,7 +528,7 @@ export default async function Page({
   const artistData = getArtist(username)
   const albumsData = getAlbums(username)
 
-  // Initiate both requests in parallel
+  // Initiate both requests concurrently
```

Comment:
This comment is not at the right level, let's move it to 527 (above getArtist) and just say `initiate requests`